### ISSUE TITLE
transmit scale link and tip

### ISSUE THREAD

**Issue by cokes518** [helium/explorer#458]

**Comment by allenan** [comment]
can you provide the content for these items?

**Comment by cokes518** [comment]
```
Transmit Scale

Transmit scale is a multiplier that is applied to rewards of any Hotspot that witness you. The scale also affects how many rewards you receive as a Challengee.

Low Transmit scale does not mean your Hotspot earnings will be low. Make sure you have a great setup so you can witness Hotspots with 1.0 Transmit scales.
```

This might be too wordy. lmk.
